pilot/cmd/pilot-discovery/main.go

```diff
@@ -76,6 +90,24 @@ var (
 				return fmt.Errorf("failed to start discovery service: %v", err)
 			}
 
+			if serverArgs.Chiron.EnableChiron {
```

Comment:
Please don't use component internal name - users don't know or care about it. 
The config must go into mesh config ( in the API ), and have a descriptive name  - like 

k8s_certificates: 
  enabled: true 
  other options if we need them...

Reply:
https://github.com/istio/api/pull/1105 is created to put the config in mesh config in the API repo. Please review it.